aws/mlcluster-vpc.template

```diff
@@ -938,3 +938,9 @@ Outputs:
           - ElasticLoadBalancer
           - DNSName
         - ':8001'
+  PrivateSubnetRouteTable:
+    Description: Private Subnet Route Table
+    Value: !GetAtt [VpcStack, Outputs.PrivateSubnetRouteTable]
```

Comment:
In outputs, we call them `* ID`, to differentiate from other identifiers such as ARN. So please add to the description.

Reply:
Same for InstanceSecurityGroup